sunflower-land/sunflower-land
# [FEAT] Pharaoh Chicken

# Description

Adds Pharaoh's treasure Mutant - Pharaoh Chicken

## How to Test

1. Add Pharaoh Chicken to Game and place on farm
2. Ensure you can dig 26 times
3. Ensure you cannot remove the placed chicken after digging 26 times

## Screenshots

<img width="436" alt="1" src="https://github.com/user-attachments/assets/527d224b-c13d-40dc-9c39-b57cea3e8144">
<img width="262" alt="2" src="https://github.com/user-attachments/assets/3fb24340-9f0d-4aca-9314-8b369665261e">


## Type of change


# How Has This Been Tested?

localhost

# Checklist:
